manuscript.md

```diff
@@ -42,7 +46,35 @@
 
 ## How are interaction strengths actually inferred? 
 
-## Could we use hypergraphs and multi-layer networks to predict more interactions? 
+## Could we use hypergraphs and multi-layer networks to predict more interactions?
+
+The web of life is built by a variety of interactions that can vary over time
+and space; these interactions include facilitation, niche construction,
+zoonoses, vector-borne diseases, among others [@Garcia-Callejas2018MulInt],
+which all share the fact that interactions between species are themselves
```

Comment:
>  which all share the fact that interactions between species are themselves are interacting

i think we should be more clear the hypergraphs are what we are using in the higher-order interaction case

and that multilayer networks are distinct from hypergraphs and used for 
- nodes are species, layers are different time points/locations (the pilosof paper referenced later) 
- nodes are spatial coordinates, layers represent dispersal between species ([example 1](https://conbio.onlinelibrary.wiley.com/doi/pdf/10.1111/cobi.12943), [example 2](https://besjournals-onlinelibrary-wiley-com.proxy3.library.mcgill.ca/doi/pdf/10.1111/1365-2435.13237))

Reply:
I have re-organized the answer into two paragraphs, to make this clearer.